src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -100,6 +76,8 @@
 
     private static final int VANITY_BLOOM_FILTER_MAX_ENTRIES = 10000000;
 
+    private static final Logger logger = LoggerFactory.getLogger(MapEntries.class);
```

Comment:
Either make this upper case or make it an instance field (instance field is probably better from a class reload POV ).

Reply:
Removed static keyword from here